tsconfig.json

```diff
@@ -56,7 +56,8 @@
       "module": "ES2020",
       "target": "ES2020",
       "esModuleInterop": true,
-      "types": ["cypress-file-upload"]
+      "types": ["cypress-file-upload"],
+      "skipLibCheck": true
```

Comment:
Why do you need this skipLibCheck ? Won't it affect other parts of our code ?

Reply:
I forgot to reply, that I don't remember exactly why it was added, but I checked if lack of it broke anything and it didn't so I deleted it :)